scipy/stats/_binomtest.py

```diff
@@ -0,0 +1,258 @@
+from collections import namedtuple
+from math import sqrt
+import numpy as np
+from scipy.optimize import brentq
+from scipy.special import ndtri
+from ._discrete_distns import binom
+
+
+ConfidenceInterval = namedtuple('ConfidenceInterval', ['low', 'high'])
+
+
+class BinomTestResult:
+
+    def __init__(self, k, n, alternative, pvalue, proportion_estimate):
+        self.k = k
+        self.n = n
+        self.alternative = alternative
+        self.proportion_estimate = proportion_estimate
+        self.pvalue = pvalue
+
+    def __repr__(self):
+        s = ("BinomTestResult("
+             f"k={self.k}, "
+             f"n={self.n}, "
+             f"alternative={self.alternative!r}, "
+             f"proportion_estimate={self.proportion_estimate}, "
+             f"pvalue={self.pvalue})")
+        return s
+
+    def proportion_ci(self, confidence_level=0.95, method='exact'):
+        """
```

Comment:
Will this documentation render somewhere?

Reply:
It will in my latest update: https://22516-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.stats.BinomTestResult.proportion_ci.html